src/objective-c/tests/InteropTests.m

```diff
@@ -152,6 +145,59 @@ - (void)testLargeUnaryRPC {
   [self waitForExpectationsWithTimeout:TEST_TIMEOUT handler:nil];
 }
 
+// TODO (mxyan): Do the same test for chttp2
```

Comment:
The responsibility of this layer is to send all ops in one batch, and that's what we have to test. Not sure of what facilities the C layer has, to help with that. Whether Cronet is involved should be transparent here.